UniMath/README.md

```diff
@@ -55,6 +55,8 @@ less fragile and to make the files have a more uniform and pleasing appearance.
 * Start all proofs with `Proof.` on a separate line and end it with
   `Defined.` on a separate line, as this makes it possible for us to generate
   HTML with expansible/collapsible proofs.
+* Use `Definition` for definitions and, `Lemma` or `Theorem` for
```

Comment:
It's too close to a tautology to say "Use `Definition` for definitions", so let's write things in the other order:

"Use Lemma, Proposition, or Theorem for proofs of propositions; for defining elements of types that are not propositions, use Definition."

Reply:
I like Dan's suggestion so I'll add it. I agree that it might make sense to use Definition for some propositions (like `iscontr`), but I'm not sure if we need to be this detailed in the style guide.